src/LaravelRequestDocsMiddleware.php

```diff
@@ -28,9 +28,15 @@ class LaravelRequestDocsMiddleware extends QueryLogger
     public function handle(Request $request, Closure $next): Response
     {
         if (!$request->headers->has('X-Request-LRD') || !config('app.debug')) {
-            return $next($request);
+            $response    = $next($request);
+            $jsonContent = json_encode([
+                'data' => $response->getData()
+            ]);
+            $response->setContent($jsonContent);
+            return $response;
```

Comment:
Note:

```
There was 1 failure:

1) Rakutentech\LaravelRequestDocs\Tests\LaravelRequestDocsMiddlewareTest::testMissingLRDHeader
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
 '{\n
-    "test": true\n
+    "data": {\n
+        "test": true\n
+    }\n
 }'
```

Also note: https://github.com/rakutentech/laravel-request-docs/pull/177/files#r1124390042

Reply:
Replied at https://github.com/rakutentech/laravel-request-docs/pull/177/files#r1124440150

Reply:
Actually this is a good test.
We don't want accidentally create a security risk.